.github/workflows/unit_tests_gpu.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: GPU pip installs
         run: |
           pip install accelerate
-          CMAKE_ARGS="-DLLAMA_CUBLAS=on" pip install "llama-cpp-python!=0.2.58"
+          CMAKE_ARGS="-DLLAMA_CUBLAS=on" pip install "llama-cpp-python==0.2.59"
```

Comment:
I'm considering going back to the previous version. For `llama-cpp-python`, v0.2.59 is the best I've seen. It passes on 3.8, 3.9 and 3.12. But fails on 3.10 and 3.11.